src/wrapt/__init__.py

```diff
@@ -28,3 +28,32 @@
 # bundled implementation here in case any user of wrapt was also needing it.
 
 from .arguments import formatargspec
+
+__all__ = (
+    "ObjectProxy",
+    "CallableObjectProxy",
+    "FunctionWrapper",
+    "BoundFunctionWrapper",
+    "PartialCallableObjectProxy",
+
+    "resolve_path",
+    "apply_patch",
+    "wrap_object",
+    "wrap_object_attribute",
+    "function_wrapper",
+    "wrap_function_wrapper",
+    "patch_function_wrapper",
+    "transient_function_wrapper",
+
+    "WeakFunctionProxy"
+
+    "adapter_factory",
+    "AdapterFactory",
+    "decorator",
+    "synchronized",
+
+    "register_post_import_hook",
+    "when_imported",
+    "notify_module_loaded",
```

Comment:
is this in the api? it's not included in the docs

Reply:
If you are specifically talking about `notify_module_loaded` then yes, it looks like it was probably a mistake that it was being imported by the `__init__.py` file as I think it is an internal function.

Reply:
removed from `__all__`

Reply:
Am actually going to add this back. The `notify_module_loaded()` function was a part of the public API originally intended in https://peps.python.org/pep-0369/ and which this functionality was based. As much as may not be used, it has been there for quite a while as a public API in `wrapt`.